src/ua/hillel/FindSymbolOccurance.java

```diff
@@ -0,0 +1,23 @@
+package ua.hillel;
+
+public class FindSymbolOccurance {
+    public static void main(String[] args) {
+
+        findSymbolOccurrance();
+    }
+
+    public static void findSymbolOccurrance() {
```

Comment:
Your method should return `int` - the number of occurrences of a symbol in a given string.
Method `findSymbolOccurrance()` should accept 2 parameters: `String source, char symbol` and its logic should be updated accordingly